src/main/java/org/jenkinsci/tracey/RabbitMQConnectionHolder.java

```diff
@@ -0,0 +1,16 @@
+package org.jenkinsci.tracey;
+
+import net.praqma.tracey.broker.impl.rabbitmq.RabbitMQConnection;
+
+public class RabbitMQConnectionHolder {
+
+    private static RabbitMQConnection connection;
```

Comment:
Every Jenkins instance has only one global connection to RabbitMQ server. This connection has only one channel. But every job has unique consumer/queue within the channel. We just need to ensure we cancel a subscription.